exercise_04.py

```diff
@@ -0,0 +1,24 @@
+def found_pi(number):
+    formula = 0
+
+    for i in range(1, number + 1):
+        formula += (-1) ** ( i + 1) / (2 * i - 1)
+        pi = 4 * formula
+
+    return f"{pi:.4f}"
+
+
+def print_pi_values_table(n):
+    print(f"{'i':<25}m(i)")
+    to_print = ""
+    for i in range(1, n + 1, 100):
+        to_print += f"\n{i:<25}{found_pi(i)}"
+        
+    return to_print
+
+def main():
+    print(print_pi_values_table(901))
```

Comment:
If you make the other fix I suggest in this file, you will just have to call `print_pi_values_table`. Don't you see that it is quite redundant to call `print(print_whatever_you_want_to_print))`?